Assets/Scripts/UI/StartMenu/RaceSelector.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class RaceSelector : MonoBehaviour
+{
+    private int trackSelector = 0;
+    private static int shipSelector = 0;
+    public void StartGame() 
+    {
+        SceneManager.LoadScene(1);
+    }
+
+    public void setTrack(int Selector)
+    {
+        trackSelector = Selector;
+        Debug.Log("set track " + trackSelector);
+    }
+    
+    public void setShip(int Selector)
+    {
+        shipSelector = Selector;
+        Debug.Log("set ship " + shipSelector);
+    }
+
+    public void StartRace()
```

Comment:
You seem to have 2 versions of the same function: StartRacer and StartGame
Also, whenever you can try to use the buildIndex to load a scene since it is slightly faster in the backend for unity